hybrid/src/main/java/jetbrains/jetpad/hybrid/HybridEditorSpecUtil.java

```diff
@@ -22,8 +22,12 @@
 public final class HybridEditorSpecUtil {
 
   public static ParsingContextFactory getParsingContextFactory(SimpleHybridEditorSpec<?> simpleHybridEditorSpec) {
-    ParsingContextFactory parsingContextFactory;
     CommentSpec commentSpec = simpleHybridEditorSpec.getCommentSpec();
+    return getParsingContextFactory(commentSpec);
+  }
+
+  public static ParsingContextFactory getParsingContextFactory(CommentSpec commentSpec) {
+    ParsingContextFactory parsingContextFactory;
     String commentPrefix = commentSpec.getCommentPrefix();
     if (commentPrefix == null) {
       parsingContextFactory = new SimpleParsingContextFactory();
```

Comment:
I'm a strong believer in one exit point. I prefer to have one return statement at the end. 
Sometimes it's harder to refactor methods with multiple returns.  Return in the middle is kind of a goto statement. If there are too many of them it could be difficult to follow the logic. You can easily live without them. There are languages that don't allow multiple returns.